components/clp-package-utils/clp_package_utils/scripts/archive_manager.py

```diff
@@ -0,0 +1,206 @@
+import argparse
+import logging
+import subprocess
+import sys
+from pathlib import Path
+
+from clp_py_utils.clp_config import StorageType
+
+from clp_package_utils.general import (
+    CLP_DEFAULT_CONFIG_FILE_RELATIVE_PATH,
+    dump_container_config,
+    generate_container_config,
+    generate_container_name,
+    generate_container_start_cmd,
+    get_clp_home,
+    load_config_file,
+    validate_and_load_db_credentials_file,
+)
+
+# Command/Argument Constants
+FIND_COMMAND = "find"
+DEL_COMMAND = "del"
+BY_IDS_COMMAND = "by-ids"
+BY_FILTER_COMMAND = "by-filter"
+BEGIN_TS_ARG = "--begin-ts"
+END_TS_ARG = "--end-ts"
+DRY_RUN_ARG = "--dry-run"
+
+logger = logging.getLogger(__file__)
+
+
+def _validate_timestamps(begin_ts, end_ts):
+    if begin_ts > end_ts:
+        logger.error("begin-ts must be <= end-ts")
+        return False
+    if end_ts < 0 or begin_ts < 0:
+        logger.error("begin-ts and end-ts must be non-negative.")
+        return False
+    return True
+
+
+def main(argv):
+    clp_home = get_clp_home()
+    default_config_file_path = clp_home / CLP_DEFAULT_CONFIG_FILE_RELATIVE_PATH
+
+    # Top-level parser and options
+    args_parser = argparse.ArgumentParser(description="View or delete archives.")
```

Comment:
Thinking if there's better descriptions..

What I can think of is "View the list of or delete compressed archives." but there should be better ones